Sources/SwiftBundler/Configuration/ProjectConfiguration.swift

```diff
@@ -3,85 +3,61 @@ import Parsing
 import Version
 import ErrorKit
 
+@Configuration(overlayable: true)
 struct ProjectConfiguration: Codable {
+  // TODO: Fix this once we can detect static properties
```

Comment:
Need to check what this was about again. Might've been resolved already.

Reply:
I believe it may have been referring to the fact that my original macro implementation was including static properties in the schema or something along those lines at least.